packages/eslint-config-khan/index.js

```diff
@@ -24,6 +24,24 @@ module.exports = {
         react: {
             version: "16.4",
         },
+        "import/parsers": {
+            "@typescript-eslint/parser": [".ts", ".tsx"],
+        },
+        "import/resolver": {
+            typescript: {
+                project: [
+                    "packages/*/tsconfig.json",
```

Comment:
Does the `packages` pattern work on other repos? And in webapp?

Reply:
Thanks for catching that.  That is definitely repo specific.  I'll move that back to .eslintrc.js and update the README.md to include a note about this.